aws/ec2/README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  <img src="https://secrethub.io/img/integrations/aws-ec2/github-banner.png?v1" alt="AWS EC2 + SecretHub" width="390">
+</p>
+<br/>
+
+<p align="center">
+  <a href="https://secrethub.io/docs/guides/aws-ec2/"><img alt="View Docs" src="https://secrethub.io/img/buttons/github/view-docs.png?v2" height="28" /></a>
+</p>
+<br/>
+
+# Deploying to AWS EC2 using Terraform
```

Comment:
This one is a bit verbose, we could do:
```suggestion
# Deploy to EC2 using Terraform
```

Reply:
Or leave it out entirely, looks pretty clean:

![image](https://user-images.githubusercontent.com/7430639/85003945-36548000-b157-11ea-924e-7e312da11fed.png)